modules/tinymce/src/plugins/advlist/main/ts/Plugin.ts

```diff
@@ -14,6 +14,9 @@ export default () => {
     if (editor.hasPlugin('lists')) {
       Buttons.register(editor);
       Commands.register(editor);
+    } else {
+      // eslint-disable-next-line no-console
+      console.error('Please use the Lists plugin together with the AdvList plugin.');
```

Comment:
I missed this earlier, but since we are using formal names this should be `Advanced List plugin`, not the shorthand version sorry.

Reply:
NP. Done!